tests/fixtures/tools/README.md

```diff
@@ -0,0 +1 @@
+The `tools` directory contains dummy mock tests designed for version testing of third-party tools. It also includes examples of faulty tools that are either not executable or do not provide version information.
```

Comment:
"mock tests" --> "mock tools"?

Also "dummy mock" is redundant, just "mock"?

Reply:
Silly me. That should be now fixed! Let me know if there is anything else that you would like to add and/or change :)